doc/languages-frameworks/vim.section.md

```diff
@@ -154,6 +154,43 @@ in
 }
 ```
 
+### Specificities for some plugins
+#### Tree sitter
+
+By default `nvim-treesitter` encourages you to download, compile and install
+the required tree-sitter grammars at run time with `:TSInstall`. This works
+poorly on NixOS.  Instead, to install the `nvim-treesitter` plugins with a set
+of precompiled grammars, you can use `nvim-treesitter.withPlugins` function:
+
+```nix
+{ pkgs, ... }:
+{
+  environment.systemPackages = [
```

Comment:
I would avoid just write `(pkgs.neovim.override { ... })` here, instead of the full configuration: all those closing parenthesis make this looks more complicated than it really is.

Reply:
well it's mimicked on the previous example, so that it looks like all those example use the same mechanisms instead of looking radically different.

Reply:
The first 5 snippets in the "Vim" sections show just the package override...

Reply:
done